library/junos_install_config

```diff
@@ -250,11 +261,21 @@ def _load_via_netconf(module):
         logging.error(msg)
         module.fail_json(msg=msg)
 
-    logging.info("connecting to host: {0}@{1}:{2}".format(args['user'], args['host'], args['port']))
+    if args['ignore_warning'] in BOOLEANS + ["True", "False"]:
```

Comment:
I was recommending `+ "none"` because `module.booleans()` accepts `None` as a valid value. However, `module.booleans()` just returns the unmodified `None` value in that case, so you are correct that it's not necessary. I'm now fine with the code as it is.